### ISSUE TITLE
BitmapFont -> format library

### ISSUE THREAD

**Issue by bendmorris** [Beeblerox/BitmapFont#25]
What do you think of pushing the format parsing and possibly layout logic (not rendering code) to https://github.com/HaxeFoundation/format? I know some of this originated in HaxePunk, it would be great to share your additions and improvements.

**Comment by Beeblerox** [comment]
Agree. It would be really good. Need some time to think about this, about how to reorganize code for pull request
